pkg/registries/registries.go

```diff
@@ -96,27 +110,46 @@ func mergedMirrorSets(icspRules []*apioperatorsv1alpha1.ImageContentSourcePolicy
 }
 
 // EditRegistriesConfig edits, IN PLACE, the /etc/containers/registries.conf configuration provided in config, to:
-// - Mark whole registries in insecureRegistries as insecure (TLS is not required, and TLS certificate verification is not required when TLS is used)
-// - Mark whole registries in blockedRegistries as blocked (any attempts to access them fail)
+// - Mark whole registries or wildcard entries in insecureRegistries as insecure (TLS is not required, and TLS certificate verification is not required when TLS is used)
+// - Mark whole registries or wildcard entries in blockedRegistries as blocked (any attempts to access them fail)
```

Comment:
After https://issues.redhat.com/browse/OCPNODE-461 , this needs to also accept host[:port]/[namespace/…/]repo.

The code is fine as is, so let’s just document it that way.

Maybe
```suggestion
// - Mark scopes in insecureScopes as insecure (TLS is not required, and TLS certificate verification is not required when TLS is used)
// - Mark scopes in blockedScopes as blocked (any attempts to access them fail)
```
\+ document “scopes” below, e.g. the way `scopeIsNestedInsideScope` does, and rename `insecureScopes`/`blockedScopes` throughout.

Reply:
Fixed